src/actions/noting/wrap-in-note.js

```diff
@@ -1,19 +1,19 @@
+var assign = require('lodash.assign');
```

Comment:
It is, but not in Babel without that polyfill unfortunately.

This polyfill is loaded automatically in some circumstances (such as within a Node context) but not in others. Also it doesn't allow it being required more than once. Making this work for the tests as well as the build isn't worth the hassle I think. A real shame though.